### ISSUE TITLE
Can not import Popover from 'carbon-components-react' '

### ISSUE THREAD

**Issue by RadekKpc** [carbon-design-system/carbon#8592]
I had to manually add this code to the node_modules/carbon-components-react/es/index.js

![Zrzut ekranu 2021-05-6 o 18 43 47](https://user-images.githubusercontent.com/48159370/117335187-280c7100-ae9b-11eb-9460-3a9ccefaf4d3.png)

It was due to following error when import:

```javascript
import {Popover, PopoverContent} from 'carbon-components-react'
```
![Zrzut ekranu 2021-05-6 o 18 47 14](https://user-images.githubusercontent.com/48159370/117335635-a832d680-ae9b-11eb-92dc-7936f709ab4e.png)

Can I request for adding {Popover, PopoverContent} to the export path?

**Comment by RadekKpc** [comment]
I made this on my own. https://github.com/carbon-design-system/carbon/issues/8592 
Coud somebody review it?

**Comment by joshblack** [comment]
Hi there @RadekKpc! 👋 

This component is intentionally not exported as it is currently under development. We'll make sure to add it to the public entrypoint when it's ready or when we're looking to get feedback on it!

**Comment by RadekKpc** [comment]
Is there any workaround to use it anyway?

**Comment by joshblack** [comment]
@RadekKpc unfortunately not since the work is not finished 😞  Sorry about that! We're actively working on this component along with a couple others so definitely be on the lookout for them in the near future 👍

**Comment by RadekKpc** [comment]
@joshblack Ok, thank you for the quick replies :) Good luck!
